### ISSUE TITLE
Organizations works per year panel should have no optimization

### ISSUE THREAD

**Issue by fnielsen** [fnielsen/scholia#122]
```
    SELECT DISTINCT ?year ?work ?organization WHERE {
      hint:Query hint:optimizer "None".
      VALUES ?organization {  wd:Q1269766  wd:Q193196  wd:Q49112  }
```

**Comment by fnielsen** [comment]
It is ineffecient with https://tools.wmflabs.org/scholia/organizations/Q216273,Q34433,Q49112

**Comment by fnielsen** [comment]
Fixed with 6a9e7c4ae45bcf57fc5e84f567be11c8cefc417d
